src/sdlc.md

```diff
@@ -143,6 +148,11 @@ You may deploy to multiple betanets if needed, for example to rehearse an upgrad
 
 ### Step 4a: Publish Governance proposal
 
+> [!IMPORTANT]
+> Final betanet testing must be complete before the governance proposal is published.
```

Comment:
nit to make this a bit more generic, since some upgrades/features may only need an alphanet
```suggestion
> Final alphanet and betanet testing must be complete before the governance proposal is published.
```